vmm/src/cpu.rs

```diff
@@ -700,6 +720,8 @@ impl CpuManager {
             proximity_domain_per_cpu,
             affinity,
             dynamic,
+            #[cfg(target_arch = "x86_64")]
+            cpu_vendor,
```

Comment:
Rather than storing this you could store `hypervisor` instead and remove `hypervisor_type`

Reply:
Yeah, working on that. I was just fixing a broken merge. A commit to implement your suggestion is building right now.

Thanks

Reply:
It should be ready for the next round now. Thanks

Reply:
Thanks for the review and hints, Robert!